src/features/user/Profile/components/RedeemModal.tsx

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 import styles from '../styles/RedeemModal.module.scss';
-import Modal from '@material-ui/core/Modal';
-import Backdrop from '@material-ui/core/Backdrop';
-import Fade from '@material-ui/core/Fade';
+import Modal from '@mui/material/Modal';
```

Comment:
Slight differenct in font style/size for Modal, especially obvious for heading.

Reply:
Fixed Here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1467/commits/899022f498edbef1e2c102fc2fae950fb381618a

Reply:
Resolved in #1467